src/libcore/future/future.rs

```diff
@@ -23,7 +23,7 @@ use crate::task::{Context, Poll};
 /// When using a future, you generally won't call `poll` directly, but instead
 /// `await!` the value.
 #[doc(spotlight)]
-#[must_use = "futures do nothing unless polled"]
+#[must_use = "futures stay idle unless you `.await` them"]
```

Comment:
This is nit-picking, but "stay idle" is indicative of a process to me. "futures are not executed" or "futures do nothing", etc. seems clearer to me. What do you think @Centril?

Reply:
+1

Reply:
I don't feel strongly so let's stick with `do nothing` then.

@Schultzer Can you please adjust `stay idle` to `do nothing` (but keep `or poll`)?

Reply:
done